insolar/flow/flow.go

```diff
@@ -23,14 +23,19 @@ import (
 )
 
 // Handle is a one-function synchronous process that can call routines to do long processing.
-//
 // IMPORTANT: Asynchronous code is NOT ALLOWED here.
+// To create a new Handle of a given message use handler.NewHandler procedure.
+// After creating a Handle you can register it in MessageBus like this:
+// `h.Bus.MustRegister(insolar.TypeGetObject, createdHandle.WrapBusHandle)`
+// You can find an example in insolar/ladger/artifactmanager/handler.go
 type Handle func(context.Context, Flow) error
 
 // MakeHandle is a function that constructs new Handle.
 type MakeHandle func(bus.Message) Handle
 
 // Procedure is a task that can execute itself.
+// Procedure is supposed to be executed in one pulse. If you have multiple steps that can be executed
```

Comment:
"supposed to be executed without pulse interruptions". Currently, sounds like it **must** complete within single pulse.

Reply:
OK, the wording changed.